lib/llm/response/chunk.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module LLM
+  class Response::Chunk < Response
```

Comment:
Should we inherit from `Response::Completion` ? It is still a Completion response, right ?